src/config/config.js

```diff
@@ -21,20 +23,66 @@ const isPhaseZero = 0 === phase
 const isPhaseOne = 1 === phase
 const isPhaseTwo = 2 === phase
 
-const alchemyKey = env.REACT_APP_ALCHEMY_KEY
 const isEToro = env.REACT_APP_ETORO === 'true' || env.REACT_APP_NETWORK === 'etoro'
 const ipfsGateways = env.REACT_APP_IPFS_GATEWAYS || 'https://cloudflare-ipfs.com/ipfs/{cid},https://ipfs.io/ipfs/{cid},https://{cid}.ipfs.dweb.link'
-const cryptoLiteracyEndDate = env.REACT_APP_CRYPTO_LITERACY_END_DATE || '2021-12-31'
+
+const alchemyKey = env.REACT_APP_ALCHEMY_KEY
+const network = env.REACT_APP_NETWORK || 'fuse'
+const { networkId } = contractsAddress[network]
```

Comment:
initializing network & network id here, then networkExplorerUrl (in the end)
to be used in both good wallet and api class